plugins/net_plugin/net_plugin.cpp

```diff
@@ -497,10 +495,10 @@ namespace eosio {
       void transaction_ack(const std::pair<fc::exception_ptr, packed_transaction_ptr>&);
       void on_irreversible_block( const block_state_ptr& block );
 
-      void on_hs_proposal_message( const hs_proposal_message_ptr& msg );
-      void on_hs_vote_message( const hs_vote_message_ptr& msg );
-      void on_hs_new_view_message( const hs_new_view_message_ptr& msg );
-      void on_hs_new_block_message( const hs_new_block_message_ptr& msg );
+      void on_hs_proposal_message( const hs_proposal_message& msg );
+      void on_hs_vote_message( const hs_vote_message& msg );
+      void on_hs_new_view_message( const hs_new_view_message& msg );
+      void on_hs_new_block_message( const hs_new_block_message& msg );
```

Comment:
Same for these. We only need one `on_hotstuff_message` which takes a `std::variant`

Reply:
Done